src/state/CalculationLogics.ts

```diff
@@ -409,3 +409,30 @@ export function isPeriodToday(cycles: Cycle[]) {
 
   return dayOfCycle <= cycles[0].periodLength;
 }
+
+export function isMarkedFutureDays(periodDays: string[]) {
+  if (periodDays.length === 0) {
```

Comment:
You dont need this check because if period Days is empty then `includes` check below will not pass

Reply:
Removed